packages/create-schemas/package.json

```diff
@@ -45,8 +48,8 @@
         "chokidar": "3.6.0",
         "commander": "12.1.0",
         "kleur": "4.1.5",
-        "openapi-typescript": "7.0.0-rc.0",
```

Comment:
just out of curiosity, what is `-rc.0` in this context?

Reply:
`rc.0` = Release Candidate #0

It's basically a prerelease version of `7.0.0`.